swagger.yaml

```diff
@@ -124,6 +134,162 @@ paths:
           description: "Internal Server Error"
           schema:
             $ref: "#/definitions/500Response"
+  /users:
+    post:
+      tags:
+        - User
+      summary: Register user into system
+      parameters:
+        - name: createUser
+          in: body
+          description: Create user
+          schema:
+            $ref: "#/definitions/userCreated"
+      responses:
+        "201":
+          description: Created
+          schema:
+            $ref: "#/definitions/userCreated"
+        "500":
+          description: "Internal Server Error"
+          schema:
+            $ref: "#/definitions/500Response"
+
+    patch:
+      tags:
+        - User
+      summary: user updated into system
```

Comment:
This should be the the description of the API, The action which we're performing not the response.
```
Update user
```